src/app/themes.tsx

```diff
@@ -3,25 +3,65 @@
 import GlobalStyle from "@/app/global_styles";
 import isPropValid from "@emotion/is-prop-valid";
 import { useServerInsertedHTML } from "next/navigation";
-import React, { useState } from "react";
+import React, { createContext, useState } from "react";
 import { ServerStyleSheet, StyleSheetManager, ThemeProvider } from "styled-components";
 
-export const currentTheme = {
-    foregroundColor: "#000",
-    backgroundColor: "#DDD",
-    fillColor: "lightgreen",
-    fonts: ["Comic Sans MS", "Comic Sans", "cursive"],
+export type CustomTheme = {
+    foregroundColor: string;
+    backgroundColor: string;
+    primaryForegroundColor: string;
+    primaryBackgroundColor: string;
+    secondaryForegroundColor: string;
+    secondaryBackgroundColor: string;
+    accentBackgroundColor: string;
+    accentForegroundColor: string;
+    surfaceForegroundColor: string;
+    surfaceBackgroundColor: string;
+    errorColor: string;
+};
+
+export const lightTheme: CustomTheme = {
```

Comment:
not sure why all of these are exported - are they used anywhere else?

Reply:
fixed